source/aggregation-pipeline-builder.txt

```diff
@@ -154,18 +150,31 @@ your chosen view mode:
                
                .. figure:: /images/compass/agg-builder-exclude-stage-example.png 
                   :alt: Aggregation Builder exclude stage example
+         
+         .. step:: Add additional pipeline stages.
+            
+            To add an additional pipeline stage after your last aggregation 
+            stage, click :guilabel:`Add Stage`. To add an aggregation stage 
+            before your most recently added stage, click the :guilabel:`+` icon 
+            above the stage card. 
+            
+            Repeat steps 3 and 4 for each additional stage.
 
-         .. step:: Click :guilabel:`Run` at the top right of the pipeline 
-            builder. |compass-short| returns your results in the document view. 
+            .. note:: 
+
+               You can change the order of pipeline stages by dragging 
+               the header of each stage card.
+
+         .. step:: Run the pipeline. 
             
-            .. warning:: 
-               
-               When you click :guilabel:`Run`, the aggregation runs against the 
-               entire collection.
+            Click :guilabel:`Run` at the top right of the pipeline 
+            builder. |compass-short| returns your results in the document view. 
 
-         .. step:: To edit the pipeline, click :guilabel:`Edit` at the top of 
-            your pipeline toolbar to return to the aggregation pipeline builder
-            and make changes to your pipeline. 
+         .. step:: Edit your pipeline. 
```

Comment:
[note, non-blocking] I don't think this step is actually definitely part of creating an aggregation pipeline! Consider making a ticket to separate this step into its own task.

Reply:
Noted ✅ 

I was debating between keeping this in here (from the original copy) or taking it out, too. Decided to keep it in because editing can be a natural "next step" for users after running the pipeline, but what you said makes sense!